celer/tests/debug_enet.py

```diff
@@ -31,15 +31,14 @@
 w = coefs_sk[:, 0]  # also fails with coefs[:, 0]
 alpha = alphas[0]
 
-theta = (X @ w - y) / n_samples
-dnorm = norm(X.T @ theta + alpha * (1 - l1_ratio) * w, ord=np.inf)
+theta = -(X @ w - y) / n_samples
+dnorm = norm(X.T @ theta - alpha * (1 - l1_ratio) * w, ord=np.inf)
```

Comment:
When putting a minus here in front of ``w``, we should consider ``theta = -(X @ w - y) / n_samples`` and changes the ``+ n_samples * theta`` with a ``- n_samples * theta`` , which is equivalent to the formula you suggested in your last commit **script to debug: our formula for enet seems incorrect**

Reply:
agreed!